highfidelity/hifi
# Fix SpinBox.

Fix variouse issues with spinbox
f.ex: https://highfidelity.manuscript.com/f/cases/14356/Developer-Network-Networking-Listening-Port-is-broken

Test plan:
- Open settings page with SpinBox, f.ex Avatar settings
- Edit values in spin boxes with different manner (enter from keyboard, using up/down buttons, slider)
- Make sure value is changed, saved and fits to its boudaries
